src/lib/konnectors.js

```diff
@@ -61,7 +61,9 @@ export function install (cozy, konnector, timeout = 120000) {
     if (!konnector[property]) throw new Error(`Missing '${property}' property in konnector`)
   })
 
-  const { slug, source } = konnector
+  let { slug, source } = konnector
+
+  if (source.substr(0, 3) !== 'git') source = atob(source)
```

Comment:
No chance to have an `https` here ? Perhaps look for `://` in the whole string instead.